src/components/LoginComponent.jsx

```diff
@@ -1,64 +1,74 @@
-import React from 'react';
-import { withRouter } from 'react-router-dom';
-import { Button, Container, Header, Image } from 'semantic-ui-react';
-import { SemanticToastContainer } from 'react-semantic-toasts';
+import React from "react";
+import { withRouter } from "react-router-dom";
+import { Button, Container, Header, Image } from "semantic-ui-react";
+import { SemanticToastContainer } from "react-semantic-toasts";
 
-import { signInWithEmail, firebase } from '../firebase';
-import { ToastMessage } from '../_utils/ToastMessage';
-import { validAndelaEmail } from '../_utils/validAndelaEmail';
+import { signInWithEmail, firebase } from "../firebase";
+import { ToastMessage } from "../_utils/ToastMessage";
+import { validAndelaEmail } from "../_utils/validAndelaEmail";
 
-import '../_css/LoginComponent.css';
+import "../_css/LoginComponent.css";
 
 const provider = new firebase.auth.GoogleAuthProvider();
 
 class LoginComponent extends React.Component {
-
   // checks that the user used an Andela email
-  validateUser = (result) => {
+  validateUser = result => {
     if (validAndelaEmail(result.user.email)) {
-      localStorage.setItem('token', result.credential.accessToken);
-      ToastMessage.success({ message: 'Welcome to ART' });
-      this.props.history.push('/dashboard');
+      result.user.getIdToken().then(function(idToken) {
```

Comment:
As houndCi has said `Unexpected function expression prefer-arrow-callback`. Please use arrow function as we are using ES6